asset/css/markbind.css

```diff
@@ -298,7 +298,7 @@ li.footnote-item:target {
     background-color: #eee;
 }
 
-pre {
+pre.hljs {
```

Comment:
The class `.hljs` is applied to the `<code>` inside of `<pre>`

Reply:
Fixed! thanks